docker.go

```diff
@@ -507,15 +503,28 @@ func (docker *dockerConfig) refreshImage(image string) {
 	log.Println()
 }
 
-func loginToECR(account string, region string) {
-	svc := ecr.New(ecr.Options{Region: region})
+func (docker *dockerConfig) tryLoginToECR(image string) error {
```

Comment:
I think we're doing too much in this method now.. previously this was just login to ECR. Now it's renamed with the try, but it logs something that makes sense in the context of the caller, then does the try login, then also downloads the image in there. I think it was clearer when the caller was responsible for those activities, and the login method was handling login.

Reply:
I moved back the get image in the calling function